deploifai/cloud_profile/create.py

```diff
@@ -33,31 +41,89 @@ def create(context: DeploifaiContextObj, name: str):
         click.echo(click.style("Cloud profile name taken. Existing names: ", fg="red") + ' '.join(existing_names))
         raise click.Abort()
 
-    provider = prompt(
-        {
-            "type": "list",
-            "name": "provider",
-            "message": "Choose a provider for the new cloud profile",
-            "choices": [{"name": provider.value, "value": provider} for provider in Provider]
-        }
-    )["provider"]
+    if not provider:
+        try:
+            provider = prompt(
+                {
+                    "type": "list",
+                    "name": "provider",
+                    "message": "Choose a provider for the new cloud profile",
+                    "choices": [{"name": p.value, "value": p} for p in Provider]
+                }
+            )["provider"]
+        except KeyError:
+            click.secho('Mouse click detected', fg="red")
+            raise click.Abort()
+    else:
+        provider = Provider(provider)   # Cast to Provider enum
 
     cloud_credentials = {}
     if provider == Provider.AWS:
-        cloud_credentials["awsAccessKey"] = prompt(
-            {
-                "type": "input",
-                "name": "awsAccessKey",
-                "message": "AWS Access Key ID (We'll keep these secured and encrypted)",
-            }
-        )["awsAccessKey"]
-        cloud_credentials["awsSecretAccessKey"] = prompt(
-            {
-                "type": "input",
-                "name": "awsSecretAccessKey",
-                "message": "AWS Secret Access Key (We'll keep these secured and encrypted)",
-            }
-        )["awsSecretAccessKey"]
+        s = boto3.session.Session()
+        profiles = s.available_profiles
+        profile = 'default'
+        if not profiles:
+            click.secho('No AWS credentials found. Please set this up', fg="red")
```

Comment:
`AWS credentials not found. Please set it up with "aws configure", or go to https://docs.deploif.ai/cloud-services/connect-your-account/aws to learn how to create AWS credentials for Deploifai on the dashboard.`